### PR TITLE
Remove SourcePath and add specific properties for redirection files

### PR DESCRIPTION
**Pull request by freewheel70**
[AB#595437](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/595437) 

Remove SourcePath and add specific properties for redirection files in .publish.json

### CODE REVIEW

File: src/docfx/build/redirection/RedirectionBuilder.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Diagnostics;
```
Comment (herohua) [inline src/docfx/build/redirection/RedirectionBuilder.cs:4]: Is this `using` necessary?

File: src/docfx/build/redirection/RedirectionProvider.cs [hunk 2]
```diff
@@ -154,34 +163,34 @@ public FilePath GetOriginalFile(FilePath file)
         return redirectUrls;
     }
 
-    private RedirectionItem[] LoadRedirectionModel()
+    private RedirectionItem[] LoadRedirectionModel(Dictionary<PathString, string> redirectConfigPaths)
```
Comment (herohua) [inline src/docfx/build/redirection/RedirectionProvider.cs:166]: why not adding `ConfigPath` as a property of `RedirectionItem`?

**Review by 928PJY — APPROVED** [review]
:shipit:

**Comment by yufeih** [comment]
Close this PR as we don't need these extra outputs.

**Pull request closed without merging**